hera_qm/tests/test_utils.py

```diff
@@ -404,3 +404,16 @@ def test_dynamic_slice():
     nt.assert_equal(b.shape, (1, 5))
 
     nt.assert_raises(ValueError, utils.dynamic_slice, 'foo', slice(0, None))
+
+
+def test_strip_extension():
+    path = 'goo/foo.boo/hoo/woo.two'
+    root = utils.strip_extension(path)
+    nt.assert_equal(root, path[:-4])
+
+
+def test_strip_extension_return_ext():
```

Comment:
This suggestion will make a lot more sense when you look at the following comment. It's about being pedantic with exactly what you're testing. But not really necessary either, feel free to reject.
```suggestion
def test_strip_extension_return_ext_basename()
```